app/grants/templates/grants/index.html

```diff
@@ -174,6 +170,25 @@
         document.sub_round_slug = "{{clr_round.sub_round_slug}}";
         document.customer_name = "{{clr_round.customer_name}}";
       {% endif %}
+      let base_style = {
+          bg: '/static/v2/images/bg/{{styles.bg}}',
+          size: "{{styles.bg_size}}",
+          color: "{{styles.bg_color}}",
+          background_image: "url('/static/v2/images/bg/{{bottom_back}}')"
+        };
+
+      document.all_type_styles = {{ all_styles|safe }}
```

Comment:
These template portions marked safe are only configurable by admins right? Something we'll have to remember if we ever have like self-service rounds/categories @thelostone-mc

Reply:
this seems is missing inline_css part. Why we are not doing this with vuejs?